en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
+3. In the test view that opens to the right of the page, click **GET**.
 
-3. Click **Try it out**, and then click **Execute**.
+4. Click **Try it out**, and then enter the following information in the fields that appear.
 
-The following figure shows the server response to the search results for the United States COVID-19 statistics:
+    | **Field**     | **Value**                                         |
+    |---------------|---------------------------------------------------|
+    | **country**   | `USA`                                            |
+    | **date**      | `2019`                                         |
+   
+5. Click **Execute**.
+
+    The response is displayed as follows:
 
-![Server Response](../assets/img/services/server-response.png){.cInlineImage-half}
+    ![Response in the test view](../assets/img/services/test-view-response.png){.cInlineImage-bordered}
 
-Now that you have verified that the `covid-stats` service works as expected, you can deploy it.
 
-## Step 4: Deploy the service
+## Step 7: Deploy the service
 
-To deploy the `covid-stats` service, follow this procedure:
+To deploy the `CovidStatus` service, follow the procedure below:
 
 1. Click the **Go Live** icon in the left pane.
 
-    ![Test Icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
+    ![Deploy icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
 
 2. To deploy the service, click **Deploy**.
 
-    The status of the service changes to **Deployed**, and the following logs show that the service is successfully deployed:    
+    The status of the service changes to **Deployed**, and the following logs appear to indicate that the product is successfully deployed.    
 
     ![Deployment logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
-
+    
+    In addition, the following message appears:
+    
+    ```
+    Deployed successfully. Note that the service will automatically undeploy in 12 hours.
+    ```
+   
 3. Once the service is deployed, click on **://cURL**.
 
     ![Get cURL command](../assets/img/services/copy-curl-command.png){.cInlineImage-full}
     
     Copy the cURL command that is displayed.
     
-4. Invoke the `covid-stats` service by issuing a few cURL commands using the Postman application. The responses are logged as shown below:
-    
-    ![covid-stats-log](../assets/img/services/covid-stats-log.png){.cInlineImage-half}
-    
-5. On the Choreo Console **Go Live** tab, the requests are logged as follows:
-    
-    ![Execution History](../assets/img/services/execution-history.png){.cInlineImage-half}
-    
-Now you are ready to observe the `covid-stats` service based on the statistics generated as a result of the cURL commands you issued. 
+    !!! tip
+        The cURL command will be something as given in the example below:<br/><br/>
+        `curl "https://covidstatus-johndoe-test.dv.choreo.dev/stats/USA/2019" -X GET`<br/><br/>
```

Comment:
It seems part of command is missing. it has API key too

Reply:
Updated the cURL command